src/models/days-off/enums/permit-acknowledged.enums.ts

```diff
@@ -0,0 +1,6 @@
+export enum permitAcknowledged {
+  Koordinator = 'Koordinator',
+  Struktural = 'Struktural',
+  HR = 'HR',
+  'Rekan Kerja' = 'Rekan Kerja',
```

Comment:
```suggestion
  RekanKerja = 'Rekan Kerja',
```

Mungkin sebaiknya seperti ini yah kang? 😁